counterpoint/lib/rendergroup.ts

```diff
@@ -699,7 +699,23 @@ export class MarkRenderGroup<
   count(): number {
     return this.markSet.size;
   }
-
+  /**
+   * Sets transform of an attribute to a new function, can be applied to an Attribute,
+   * a Mark, or a MarkRenderGroup
+   * @param attrToModify attribute to modify
+   * @param newFunc new function to set transform to
+   */
+  setTransform(
+    attrToModify: string,
+    newFunc: (raw: any, computeArg: any) => any
+  ): void {
+    this.forEach((m) =>
+      m.attributes[attrToModify].setTransform(
```

Comment:
Let's do this as `m.setTransform(attrToModify, newFunc)` instead of directly accessing the attributes of the mark.